dice_ml/model_interfaces/base_model.py

```diff
@@ -37,16 +38,18 @@ def load_model(self):
             with open(self.model_path, 'rb') as filehandle:
                 self.model = pickle.load(filehandle)
 
-    def get_output(self, input_instance):
-        """returns prediction probabilities for a classifier and the
-        predicted output for a regressor.
+    def get_output(self, input_instance, model_score=True):
```

Comment:
Sorry, this is a very small detail, but should the default value be False since `predict_fn` is used more often than `predict_fn_scores`?

Reply:
Initially, I had the name actual_predictions instead of model_scores. @amit-sharma had suggested model_scores. 

We cannot change the default behavior of get_output() which is to return probabilities for classification scenarios since this breaks backward compatibility with respect to earlier releases. Hence, the model_scoresis True by default.